src/TabView.tsx

```diff
@@ -141,7 +141,8 @@ export default function TabView<T extends Route>({
 
 const styles = StyleSheet.create({
   pager: {
-    flex: 1,
+    flexGrow: 1,
+    flexShrink: 1,
```

Comment:
If your PR targets only web you should check if we are currently on web. Also did you test this on iOS and Android?